packages/amplication-server/src/core/app/app.resolver.spec.ts

```diff
@@ -369,9 +386,9 @@ const findManyEnvironmentsMock = jest.fn(() => {
   return [EXAMPLE_ENVIRONMENT];
 });
 
-const mockCanActivate = jest.fn(() => true);
+const mockCanActivate = jest.fn(mockGqlAuthGuardCanActivate(EXAMPLE_USER));
 
-describe('AppResolver', () => {
+describe('AppService', () => {
```

Comment:
AppResolver was the correct one

Reply:
Weird, I probably branched from old master!

Reply:
But you haven't fixed it 😢